android/libraries/rib-base/src/main/java/com/badoo/ribs/android/recyclerview/RecyclerViewHostFeature.kt

```diff
@@ -0,0 +1,84 @@
+package com.badoo.ribs.android.recyclerview
+
+import android.os.Parcelable
+import com.badoo.mvicore.element.Bootstrapper
+import com.badoo.mvicore.element.Reducer
+import com.badoo.mvicore.element.TimeCapsule
+import com.badoo.mvicore.feature.ReducerFeature
+import com.badoo.ribs.android.recyclerview.RecyclerViewHost.Input
+import com.badoo.ribs.android.recyclerview.RecyclerViewHostFeature.State
+import com.badoo.ribs.core.routing.configuration.ConfigurationKey
+import io.reactivex.Observable
+import kotlinx.android.parcel.Parcelize
+import java.util.UUID
+
+private val timeCapsuleKey = "RecyclerViewHostFeature"
+private fun <T : Parcelable> TimeCapsule<State<T>>.initialState(): State<T> =
+    (get(timeCapsuleKey) ?: State())
+
+
+internal class RecyclerViewHostFeature<T : Parcelable>(
+    timeCapsule: TimeCapsule<State<T>>,
+    initialElements: List<T>
+) : ReducerFeature<Input<T>, State<T>, Nothing>(
+    initialState = timeCapsule.initialState(),
+    bootstrapper = BootstrapperImpl(timeCapsule.initialState(), initialElements),
+    reducer = ReducerImpl()
+) {
+    init {
+        timeCapsule.register(timeCapsuleKey) {
+            state.copy(
+                lastCommand = null
+            )
+        }
+    }
+
+    @Parcelize
+    data class State<T : Parcelable>(
+        val nextKey: Int = 1000,
```

Comment:
why 1000?)

Reply:
For debugging purposes. Otherwise has no significance.